client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1241,7 +1245,8 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
         pushRackAwareEnabled,
         userIdentifier,
         slotsAssignMaxWorkers,
-        availableStorageTypes)
+        availableStorageTypes,
+        workerStatusTracker.excludedWorkers.asScala.keys.toSet)
```

Comment:
I think we'd better add a config to enable this feature to avoid the corner case where most of the workers are excluded.

Reply:
done